src/traces/scattergl/plot.js

```diff
@@ -53,7 +53,6 @@ var exports = module.exports = function plot(gd, subplot, cdata) {
         return;
     }
 
-    var count = scene.count;
```

Comment:
Would it be better to just `var count = cdata.length` in that case - and remove all the other changes?

Reply:
Good point! @eiriklv Could you please `fetch upstream/master` on your fork and test it by opening a PR?